tests/pipeline/test_filter.py

```diff
@@ -820,3 +822,40 @@ def test_top_and_bottom_with_groupby_and_mask(self, dtype, seed):
             },
             mask=self.build_mask(permute(rot90(self.eye_mask(shape=shape)))),
         )
+
+
+class SpecificAssetsTestCase(WithSeededRandomPipelineEngine,
```

Comment:
why does this need a seeded random loader, we are not loading any values, just sids. Can we just create an engine with an exploading object for a loader?